performance.lic

```diff
@@ -0,0 +1,156 @@
+custom_require.call(%w(common common-travel))
```

Comment:
❌ Please add a documentation header with a link to elanthipedia

Reply:
Done